x-pack/plugins/security_solution/public/cases/containers/use_post_comment.tsx

```diff
@@ -80,7 +82,7 @@ export const usePostComment = (caseId: string): UsePostComment => {
       };
     },
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [caseId]
+    []
```

Comment:
why is this empty?

Reply:
we should remove the `// eslint-disable-next-line react-hooks/exhaustive-deps` each time we see one. That's a mistake from our misunderstanding of the dependencies at the beginning of hook